docs/reference/technology/firmware_update.md

```diff
@@ -86,12 +86,12 @@ Mbed CLI and the Online Compiler implement the Pelion Device Management Update s
 
 <span class="notes">**Note**: The default workflow of the development tools is intended only for development and testing purposes. It is not considered secure for production.</span>
 
-Your development tool needs to use your Device Management account's API key to call the Pelion Device Management APIs. Once it has access to the APIs, it can generate a manifest and upload the manifest and binary to the server, and deliver the manifest to a device or group of devices. For more information, see [the update API documentation](https://cloud.mbed.com/docs/latest/service-api-references/update-service.html).
+Your development tool needs to use your Device Management account's API key to call the Pelion Device Management APIs. Once it has access to the APIs, it can generate a manifest and upload the manifest and binary to the server, and deliver the manifest to a device or group of devices. For more information, see [the update API documentation](https://www.pelion.com/docs/device-management/current/service-api-references/update-service.html).
 
 <span class="notes">**Note**: The tools currently support the update process for Device Management Client, not Device Management Client Lite.</span>
 
 **Tutorials**:
 
-- Try the firmware [update process on the Online Compiler](https://cloud.mbed.com/guides/pelion-firmware-update).
-   <span class="notes">**Note:** You first need to [use the connecting quick guide to set up the original application](https://cloud.mbed.com/guides/connect-device-to-pelion).</span>
+- Try the firmware [update process on the Online Compiler](../guides/pelion-firmware-update/).
```

Comment:
No, of course not. They're relative.

Reply:
(Or at least, it doesn't work when I try it)

Reply:
No, but both quick starts are hosted on the OS docs site. Are relative links still unsuitable since they're outside the docs set itself? 🤔

Reply:
Try it. You have a build of that branch.

Reply:
Right. It worked for me, but went through a redirect because the link was to cloud.mbed.com. Odd. In any case, Pekka set me straight: it doesn't matter that the material is all hosted on the same domain; what matters is that the sources are in different repositories. Which makes total sense. So even though they're on the same product's (os.mbed.com) site, the quick connects still need absolute links. Mind blown.

Reply:
No, it doesn't matter what we keep the source. It matters where we publish it.